package.json

```diff
@@ -7,13 +7,17 @@
   },
   "main": "index.js",
   "scripts": {
+    "examples": "npm-run-all --serial examples:*",
+    "examples:microsoft": "./bin/fastatic examples/microsoft.github.io-master/ --dest dist/examples/microsoft/",
+    "examples:react": "./bin/fastatic examples/react-gh-pages/ --dest dist/examples/react/",
+    "examples:bootstrap": "./bin/fastatic examples/bootstrap-gh-pages/ --dest dist/examples/bootstrap/",
+    "postexamples:react": "node ./scripts/post-react.js",
+    "build": "npm run examples && npm run build:readme",
+    "build:readme": "marked README.md -o 'dist/index.html'",
```

Comment:
Relative links like `CONTRIBUTING.md` should be prefixed with `https://github.com/voorhoede/fastatic/blob/master/`.